files/chance/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "chance",
+  "repo": "victorquinn/chancejs",
+  "files": {
+    "basePath": "dist/"
```

Comment:
There is no dist/ directory here https://github.com/victorquinn/chancejs/tree/0.7.6
The bot will fail